scripts/deploy-ebpf.ps1.in

```diff
@@ -234,22 +237,24 @@ if ($vm -eq $null) {
    foreach ( $file in $built_files ) {
       $source_path = "$build_directory\$file"
       $destination_path = "$destination_directory\$file"
+      $destination_full_dir = Split-Path $destination_path
       Write-Host " $source_path -> $destination_path"
-      Copy-Item "$source_path" -Destination "$destination_path"
-      if (! $?) {
-         exit 1
+      if (! (Test-Path $destination_full_dir)) {
+         New-Item -Type Directory $destination_full_dir -ErrorAction Stop | Write-Verbose
       }
+      Copy-Item "$source_path" -Destination "$destination_path" -ErrorAction Stop
    }
 
    Write-Host "Copying files from `"$source_directory`" to `"$destination_directory`""
    foreach ( $file in $source_files ) {
       $source_path = "$source_directory\$file"
       $destination_path = "$destination_directory\$file"
+      $destination_full_dir = Split-Path $destination_path
```

Comment:
```suggestion
      $destination_full_directory = Split-Path $destination_path
```
[Coding convention in this project](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#coding-conventions) is to avoid abbreviations.  E.g., line 248 spells out directory.